src/components/Index.vue

```diff
@@ -1,169 +1,220 @@
+            </div>
           </div>
         </div>
-
-        <div class="col-lg-4">
-          <div :style="'background: url(' +require('@/assets/images/vector2.png')">
-            <p>Report A Sighting</p>
+      </div>
+    </div>
+    <div class="mpa-into-continued">
+      <div class="container">
+        <div class="mpa-intro-text">
+          <h1 align="center">
+            Let's Bring Them Back!
+          </h1>
+          <p>Some missing people you will have heard of, but many more you won't. For their families, life can feel like a desperate
+            and unbearable struggle as they wait for days or even years. That's why we are heare for missing persons and
+            their familie, 24 hours a day, 365 days a year, to provide free and confidential support, help us report a sighting,
+            search our database, cross check information, query the database.</p>
+        </div>
+
+        <div class="mpa-list">
+          <div class="mpa-list-guide" v-for="(latestMissing, index) in missing_people" :key="index">
+            <div class="latest-missing" :style="'background-image: url(' + latestMissing.picture + ');'">
+              <div class="latest-missing-bottom-info">
+                <div class="name-and-age">
+                  <h4>{{latestMissing.name}}</h4>
+                  <span class="age-holder">{{latestMissing.age}}</span>
+                </div>
+                <div class="latest-missing-person-location">
+                  <div>Missing Since:
+                    <small>{{latestMissing.missingSince}}</small>
+                  </div>
+                  <div>Last Seen:
+                    <small>{{latestMissing.lastSeen}}</small>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div align="center">
+            <br>
+            <button type="submit" class="btn btn-default btn-colored">LOAD MORE</button>
           </div>
         </div>
-
-        <div class="col-lg-4">
-          <div :style="'background: url(' +require('@/assets/images/vector3.jpg')">
-            <p>Donate or Support Us</p>
+      </div>
+    </div>
+    <div class="container">
+      <div class="mpa-call-to-action-group">
+        <h3 align="center">NEW HERE?</h3>
+        <div class="col-md-4">
+          <div class="cta-item-1">
           </div>
+          <a href="#">Who we are</a>
+        </div>
+        <div class="col-md-4">
+          <div class="cta-item-2"></div>
+          <a href="#">Report a sighting</a>
+        </div>
+        <div class="col-md-4">
+          <div class="cta-item-3"></div>
+          <a href="#">Donate or support us</a>
         </div>
-
       </div>
-
     </div>
   </div>
 </template>
 
 <script>
+  import countryStates from "@/utils/geolocation";
+
   export default {
-    name: 'Index'
+    name: "Index",
+    data() {
+      return {
+        countryStates,
+        latest_missing_people: [{
```

Comment:
Is it possible to have this data in a json file and then import it here just to minimise space.